Sources/App/Models/Release.swift

```diff
@@ -12,7 +12,7 @@ struct Release: Codable, Equatable {
 
 extension Release {
     init(from node: Github.Metadata.ReleaseNodes.ReleaseNode) {
-        description = node.description
+        description = node.descriptionHTML
```

Comment:
So this effectively changes `Version.releaseNotes` from containing plain text to html (no schema change obv), because it gets initialised from `Release.description`.

There is one other place where `version.releaseNotes` is currently being used: `PackageCollection.Package.Version.init?` where it gets assigned to `summary`.

We need to check if it's ok to change this from plain text to html. This might be exposed in Xcode and we wouldn't want to make it show HTML markup.

Reply:
I've completely missed it 😨 Thanks for the heads up

Reply:
Instead of replacing the `releaseNotes` content, I've introduced the `releaseNotesHTML` field. So, `PackageCollection` uses plain ol' markdown `releaseNotes`, and `RSSFeed` was switched to `releaseNotesHTML`